### PR TITLE
changed journald example to be full c&p ready

### PR DESCRIPTION
**Pull request by dmoessne**
- Following the current example is not working out of the box, as 
   - we use a variable in the example (${jrnl_cnf}) which is not filled when just copied over which leads to issues 
    - machineconfig example just shows an excerpt instead of a fully usable machineconfig. yaml as we do in other sections (e.g. chrony) 

improvement: 
  - use a full example of the machine.config that is ready to be copy and pasted
  - highlight, that in case we copy the example on needs to copy as well the created encoded file in, replacing the variable
  - use a proper naming as well, e.g. 99-....

I think this applies to all versions where we have this in the docs, so back to 4.2 

This worked in my tests but you may want to have eng check this before merging

@openshift/team-documentation.

### CODE REVIEW

**Comment by dmoessne** [comment]
bz 1882396

File: modules/cluster-logging-systemd-scaling.adoc [hunk 1]
```diff
@@ -75,32 +75,46 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 $ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
 ----
 
-. Create a new MachineConfig for master or worker and add the `journal.conf` parameters:
+. The following example creates a MachineConfig to change  `journald.conf` parameters for worker. You can change it to master or make an additional MachineConfig for the master role:
```
Comment (kalexand-rh) [inline modules/cluster-logging-systemd-scaling.adoc:78]: The step needs to include an explicit action. I think this would be a better change:

~~~
.  Create a new MachineConfig for master or worker and add the `journal.conf` parameters:
** To change the `journald.conf` parameters for workers, create the following MachineConfig:
+
[source,terminal}
----
Your new example
----

** To change the `journald.conf` parameters for control plane machines, create the following MachineConfig:
+
[source,terminal}
----
Whatever it is for masters
----
~~~

File: modules/cluster-logging-systemd-scaling.adoc [hunk 2]
```diff
@@ -75,32 +75,46 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
...
-<2> Specify the path to the base64-encoded `journal.conf` file.
+<1> If you want to copy and past just the content of the configuration you must replace `${jrnl_cnf}` with the output of `echo $jrnl_cnf`
+<2> Set the permissions for the `journal.conf` file. It is recommended to set `0644` permissions. 
 
 . Create the MachineConfig:
 +
 [source,terminal]
 ----
-$ oc apply -f <filename>.yaml
+$ oc apply -f 99-worker-journald-configuration.yaml
```
Comment (kalexand-rh) [inline modules/cluster-logging-systemd-scaling.adoc:117]: If you present both files, revert this change.

**Review by kalexand-rh — COMMENTED** [review]
@dmoessne, thanks for submitting this change! I have some suggestions.

File: modules/cluster-logging-systemd-scaling.adoc (hunk 1 above)
Comment (dmoessne) [reply modules/cluster-logging-systemd-scaling.adoc:78]: @kalexand-rh 
I am not against presenting both files, however to have it consistent with other parts for the doc I just presented one of them.
E.g. checking https://docs.openshift.com/container-platform/4.5/installing/install_config/installing-customizing.html we present either master, or worker or both. I am happy to contribute to overall more doc consistency, however, I am unsure which one it would be. If you say presenting both should be best practise I am going forward and add master config as well

Comment (kalexand-rh) [reply modules/cluster-logging-systemd-scaling.adoc:78]: @dmoessne, I think that if we're making it easier for the user to paste their way to success that providing both is better. (I'm not opposed to presenting one and explaining how to modify it, if you think that that's more appropriate, but the step has to contain an explicit action statement. You'd just have to say something like "Create a MachineConfig to change the `journald.conf` parameters for a machine type. You must apply the `journald.conf` parameters for each machine type. The following command creates a MachineConfig for worker machines:" And then add a note after the example that explains how to modify the MachineConfig if you want to change the control plane machines.)

**Comment by dmoessne** [comment]
@kalexand-rh would you mind checking again ?

**Comment by kalexand-rh** [comment]
@dmoessne, will you PTAL at your last commit? The change doesn't match the commit message.
